deploy/registry/local/deployment.yaml

```diff
@@ -13,6 +13,7 @@ metadata:
   labels:
     app: che-plugin-registry
   name: che-plugin-registry
+  namespace: operators
```

Comment:
Why do you force installing the plugin registry in the `operators` namespace ?

Afair this namespace doesn't exist on OpenShift.

Reply:
Let's discuss alternatives in the following thread https://github.com/che-incubator/che-workspace-crd-operator/pull/3#discussion_r363653290